examples/house-credit-default/main.py

```diff
@@ -38,140 +46,153 @@
-  tf.identity(x, name=pred_names[0])
+    pred_names = ["output"]
+    tf.identity(x, name=pred_names[0])
 
-  graph = graph_util.convert_variables_to_constants(
-      sess, sess.graph.as_graph_def(), pred_names)
+    graph = graph_util.convert_variables_to_constants(
+        sess, sess.graph.as_graph_def(), pred_names
+    )
 
-  graph = graph_util.remove_training_nodes(graph)
-  path = graph_io.write_graph(graph, ".", filename, as_text=False)
-  print('saved the frozen graph (ready for inference) at: ', path)
+    graph = graph_util.remove_training_nodes(graph)
+    path = graph_io.write_graph(graph, ".", filename, as_text=False)
+    print("saved the frozen graph (ready for inference) at: ", path)
 
 
 def print_nodes(graph):
-  """Print a list of nodes from a tf.Graph."""
-  print([n.name for n in graph.as_graph_def().node])
+    """Print a list of nodes from a tf.Graph."""
+    print([n.name for n in graph.as_graph_def().node])
 
 
 def build_model(input_shape):
-  """Build a logistic regression model with tf.keras."""
-  model = keras.Sequential([
-      layers.Dense(1, use_bias=False, activation='sigmoid',
-                   input_shape=[input_shape]),
-  ])
+    """Build a logistic regression model with tf.keras."""
+    model = keras.Sequential(
+        [
+            layers.Dense(
+                1, use_bias=False, activation="sigmoid", input_shape=[input_shape]
+            ),
+        ]
+    )
 
-  model.compile(loss='binary_crossentropy',
-                optimizer=tf.train.AdamOptimizer(),
-                metrics=['accuracy'])
+    model.compile(
+        loss="binary_crossentropy",
+        optimizer=tf.train.AdamOptimizer(),
+        metrics=["accuracy"],
+    )
 
-  return model
+    return model
 
 
 def train(train_x_df, train_y_df):
-  """Train a logistic regressor on the dataset."""
-  x = list(train_x_df.columns.values)
-  model = build_model(len(x))
+    """Train a logistic regressor on the dataset."""
+    x = list(train_x_df.columns.values)
+    model = build_model(len(x))
 
-  os.makedirs('./saved_models', exist_ok=True)
+    os.makedirs("./saved_models", exist_ok=True)
 
-  cp_callback = keras.callbacks.ModelCheckpoint(checkpoint_path,
-                                                save_weights_only=True,
-                                                save_best_only=True,
-                                                verbose=1)
+    cp_callback = keras.callbacks.ModelCheckpoint(
+        checkpoint_path, save_weights_only=True, save_best_only=True, verbose=1
+    )
 
-  # first 80 percent for training
-  train_x = train_x_df[1:246005]
-  train_y = train_y_df[1:246005]
+    # first 80 percent for training
+    train_x = train_x_df[1:246005]
+    train_y = train_y_df[1:246005]
 
-  # other 20 percent for evaluating
-  eval_x = train_x_df[246006:len(train_x_df) - 1]
-  eval_y = train_y_df[246006:len(train_y_df) - 1]
+    # other 20 percent for evaluating
+    eval_x = train_x_df[246006 : len(train_x_df) - 1]
```

Comment:
spacing around a colon seems strange :/

Reply:
i vaguely recall them having a discussion re this somewhere if you're interested